src/components/Navbar.js

```diff
@@ -3,91 +3,81 @@ import Notification, { notify } from 'react-notify-toast';
 import { withRouter, Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import {
+  Collapse,
+  Navbar as Navigation,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  UncontrolledDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap';
 import { logoutAction } from '../redux/actions/auth/authActions';
 
 export class Navbar extends Component {
+    state = {
+      isOpen: false,
+    };
+
   logoutUser = (event) => {
     event.preventDefault();
     this.props.logoutAction();
     this.props.history.push('/');
     notify.show('You Loggedout successfully', 'success', 3000);
   };
 
+  toggle = () => {
+    this.setState(prevState => ({
+      isOpen: !prevState.isOpen,
+    }));
+  }
+
   render() {
+    const { isOpen } = this.state;
+    const { loggedIn } = this.props;
     return (
       <div className="custom-bg-dark">
         <Notification options={{ zIndex: 10000, top: '200px' }} />
-        <nav className="navbar navbar-expand-lg navbar-light bg-dark custom-max-width">
```

Comment:
@AGMETEOR Include the `custom-max-width` class to the navbar. This kept the maximum navbar width to `1300px` and aligned it to the content on large screens